zuul.d/ansible-cloud-jobs.yaml

```diff
@@ -15,3 +15,23 @@
     required-projects:
       - name: github.com/ansible/ansible-zuul-jobs
     nodeset: vmware-vcsa-6.7.0-python36
+
+- job:
+    name: ansible-test-cloud-integration-vcsa
+    abstract: true
+    parent: ansible-cloud-vcsa-appliance
+    pre-run: playbooks/ansible-test-cloud-integration-base/pre.yaml
+    run: playbooks/ansible-test-cloud-integration-base/run.yaml
+    post-run: playbooks/ansible-test-cloud-integration-base/post.yaml
+    required-projects:
+      - name: github.com/ansible/ansible
+    timeout: 3600
+
+- job:
+    name: ansible-test-cloud-integration-vcsa-python36
+    parent: ansible-test-cloud-integration-vcsa
+    nodeset: vmware-vcsa-6.7.0-python36
+    vars:
+      ansible_test_python: 3.6
+      ansible_test_integration_target: zuul/vmware/vcenter_only/
+    attempts: 1
```

Comment:
remember to remove this once everything is 100%

Reply:
Done, this being said, the default '3' is totally overkill IMO.

Reply:
Agree in perfect world, this really protects on something funky happening on the network, which we do see in both vexxhost / limestone.  With out it, we'll see RETRY_LIMIT failures on jobs.... which we do still, just much lower rate as we try 3 times for jobs to run.